pwndbg/color/memory.py

```diff
@@ -18,7 +18,7 @@
 )
 
 
-def get(address, text=None, prefix=None) -> str:
+def get(address: int | str, text: str | None = None, prefix: str | None = None) -> str:
```

Comment:
Fwiw I believe we do `int(address)` because we may accept `gdb.Value`, not a `str`

Reply:
I'll fix it

Reply:
fixed!